packages/client/src/users/returns/types/getUserReturns.types.ts

```diff
@@ -0,0 +1,47 @@
+import type { CategorisedAddress, Config } from '../../../types';
+import type { Order, OrderItem, ReturnOptionType } from '../../../orders';
+import type { ReturnItem } from '../../../returns/types';
+import type {
+  ReturnPickupSchedule,
+  ReturnReference,
+  ReturnRefundPreference,
+  ReturnStatus,
+  ReturnStatusCode,
+} from '../../../returns/types/return.types';
+import type { User } from '../../authentication/types/user.types';
+
+export type UserReturns = {
```

Comment:
This type should be defined as this:

```suggestion
export type UserReturns = PagedResponse<Return>
```

This is because the `getUserReturns` endpoint returns `Return` entries just like the other endpoints that get returns (for example `getOrderReturns`)